src/modules/drive/AddMenu/AddMenuContent.jsx

```
@@ -53,6 +54,10 @@ const AddMenuContent = forwardRef(
     }
 
     const createActionOnClick = isReadOnly ? handleReadOnlyClick : onClick
+    console.log(
```

ESLint

[error] Unexpected console statement.

(no-console)